### PR TITLE
fix for drawer menu and title text on OnePlus devices

### PR DESCRIPTION
**Pull request by akinwale**

### CODE REVIEW

File: app/src/component/AppNavigator.js [hunk 1]
```diff
@@ -66,6 +66,7 @@ const discoverStack = createStackNavigator({
     navigationOptions: ({ navigation }) => ({
       title: 'Discover',
       headerLeft: menuNavigationButton(navigation),
+      headerTitleStyle: discoverStyle.titleText
```
Comment (seanyesmunt) [inline app/src/component/AppNavigator.js]: Is there a way to default to this? Instead of passing it in every time.

Comment (akinwale) [reply app/src/component/AppNavigator.js]: Unfortunately not. Have to define custom navigation options for every navigator object created.

Comment (seanyesmunt) [reply app/src/component/AppNavigator.js]: You could have a default nav options object,

``
navigationOptions: ({ navigation }) => ({ ...defaultOptions, title: ... })
```

Comment (akinwale) [reply app/src/component/AppNavigator.js]: I suppose this could work.

**Pull request merged**